google/cloud/forseti/scanner/audit/firewall_rules_engine.py

```diff
@@ -641,5 +655,19 @@ def is_rule_exists_violation(rule, policies, exact_match=True):
       bool: If the required rule is in the policies.
     """
     if exact_match:
-        return not any([policy == rule for policy in policies])
-    return not any([policy.is_equilvalent(rule) for policy in policies])
+        result = []
+        for policy in policies:
+            if policy == rule:
+                result.append(True)
+            else:
+                result.append(False)
+        final_result = not any(result)
+        return final_result
+    rule_policy_match = []
```

Comment:
Can we call this `result` and `final_result` here to be consistent?  Will the linter accept this?

Reply:
Yes, is should be fine as result[] and final_result[] already defined is in if block.